SofaKernel/framework/sofa/core/objectmodel/BaseContext.cpp

```diff
@@ -239,12 +239,26 @@ core::topology::Topology* BaseContext::getTopology() const
 {
     return this->get<sofa::core::topology::Topology>();
 }
+
 /// Mesh Topology (unified interface for both static and dynamic topologies)
 core::topology::BaseMeshTopology* BaseContext::getMeshTopology() const
 {
     return this->get<sofa::core::topology::BaseMeshTopology>();
 }
 
+/// Mesh Topology that is local to this context (i.e. not within parent contexts)
+core::topology::BaseMeshTopology* BaseContext::getLocalMeshTopology() const
+{
+    return this->get<sofa::core::topology::BaseMeshTopology>(Local);
+}
+
+/// Mesh Topology that is relevant for this context
+/// (within it or its parents until a mapping is reached that does not preserve topologies).
+core::topology::BaseMeshTopology* BaseContext::getActiveMeshTopology() const
+{
+    return this->get<sofa::core::topology::BaseMeshTopology>(Local);
```

Comment:
Is it normal that `getLocalMeshTopology` and `getActiveMeshTopology` return exactly the same?